src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -383,12 +383,12 @@ export class ManifoldPlanTable {
 
           return [
             <div
-              class="mp--cell mp--cell__bts mp--cell__thead mp--cell__thead mp--cell__th"
+              class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--THead ManifoldPlanTable__Cell--THead ManifoldPlanTable__Cell__TH"
```

Comment:
double class name `ManifoldPlanTable__Cell--THead`?

Reply:
Ha I guess my find-and-replace was perfect 😄